pymtl3/dsl/test/sim_utils.py

```diff
@@ -75,7 +75,7 @@ def simple_sim_pass( s, seed=0xdeadbeef ):
                         .replace( "(", "_" ).replace( ")", "" ) \
                         .replace( ",", "_" )
 
-        rstrs    = [ f"{x!r} = _w" for x in readers ]
+        rstrs    = [ f"{x!r} @= _w" for x in readers ] # THIS IS SLOW, NOW WE CAN HAVE BETTER MECHANISM
```

Comment:
Dumb question: what does the format string `{x!r}` do?

Reply:
repr(x)